### DISCUSSION TITLE
Dosage and metabolism

### DISCUSSION THREAD

**Discussion by desaimrunal** [Open-Systems-Pharmacology/Forum#1001] [Questions & Problems]
Although I have given input of hepatic and renal clearance, I am observing a sharp spike and increase in concentration.
What might be the reason?
![image](https://user-images.githubusercontent.com/83763402/138847322-ce9385eb-6ab4-471f-8ee4-6e58460e5d51.png)

**Comment by tobiasK2001** [comment]
A bit hard to say based on the given info. What is the unit of the x-axis? Minutes or hours? How does it look with y-axis in log scale. What is the route of application? In which species are we?

Best, Tobias

**Reply by desaimrunal** [reply]
The unit of X axis is hours. The species is Human. The dose is administered by oral route(Tablet). Reference plot as well as the simulated one is in linear scale.
I have given input of hepatic and renal clearance, do I need to give input of dissolution data and the kinetics which it follows..??

Thanking for the response..!!
Mrunal

**Reply by desaimrunal** [reply]
The unit of X axis is hours. The species is Human. The dose is administered by oral route(Tablet). Reference plot as well as the simulated one is in linear scale.
I have given input of hepatic and renal clearance, do I need to give input of dissolution data and the kinetics which it follows..??

Thanking for the response..!!
Mrunal

**Reply by prvmalik** [reply]
You will have to refine volume of distribution (perhaps via modulating logP in the parameter identification tool). This can usually only be identified using an IV dataset. Human is preferred, animal is acceptable.

With your Vd in place, you will enter the dissolution and formulation specs, and consider tuning specific intestinal permeability.

**Reply by desaimrunal** [reply]
Thank you..!! I will try with the itputs..!!
